build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/GoDriver.java

```diff
@@ -17,14 +17,17 @@
  * Created by Bar Belity on 13/02/2020.
  */
 public class GoDriver implements Serializable {
-    private static final String GO_VERSION_CMD = "version";
+    private static final List<String> GO_LIST_USED_MODULES_CMD =
+            Arrays.asList("list", "-f" ,"\"{{with .Module}}{{.Path}} {{.Version}}{{end}}\"", "all");
     private static final String GO_MOD_GRAPH_CMD = "mod graph";
+    private static final String GO_MOD_TIDY_CMD = "mod tidy";
+    private static final String GO_VERSION_CMD = "version";
     private static final String GO_LIST_MODULE = "list -m";
```

Comment:
GO_LIST_MODULE --> GO_LIST_MODULE_CMD
(for consistency with the other constants above).